public/config.js

```diff
@@ -1 +1,6 @@
-window.VOCASCAN_CONFIG = {};
+window.VOCASCAN_CONFIG = JSON.parse(`{
+    "themes": {
+        "dark": "default-themes/dark.css",
+        "light": "default-themes/light.css"
+    }
```

Comment:
The default config should be made here in `src/vocascan-frontend/src/config.js`

Reply:
Better now?

Reply:
Better now?